lib/fluent/engine.rb

```diff
@@ -111,7 +115,30 @@ def configure(conf)
       end
 
       @root_agent.configure(conf)
-      @event_router = @root_agent.event_router
+
+      begin
+        log_event_agent = @root_agent.find_label(Fluent::Log::LOG_EVENT_LABEL)
+        log_event_router = log_event_agent.event_router
+
+        # suppress mismatched tags only for <label @FLUENT_LOG> label.
+        # it's not suppressed in default event router for non-log-event events
+        log_event_router.suppress_missing_match!
+
+      rescue ArgumentError # ArgumentError "#{label_name} label not found"
+        # use default event router if <label @FLUENT_LOG> is missing in configuration
+        log_event_router = @root_agent.event_router
+      end
+
+      if Fluent::Log.event_tags.any?{|t| log_event_router.match?(t) }
+        @log_event_router = log_event_router
+
+        unmatched_tags = Fluent::Log.event_tags.select{|t| !@log_event_router.match?(t) }
+        unless unmatched_tags.empty?
+          $log.warn "match for some tags of log events are not defined (to be ignored)", tags: unmatched_tags
+        end
```

Comment:
Check tag typo is better. Currently, there is no warning with following conf.

```
<label @FLUENT_LOG>
  <match fleunt.info>
    @type stdout
  </match>
</label>
```

Log example: `There is no 'fluent' prefix <match> in <label @FLUENT_LOG>. Check typo.`